src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -35,6 +38,14 @@
       @end
 
 
+      # FIXME(hzyi): handle ListInitType, MapInitType and StructInitType correctly
```

Comment:
Instead of having this code that duplicates the one below, why don't you make `shouldImportSix` return all these code lines, and then you include them here? Above, you can just check whether the lines are non-empty.

I'm thinking something like this, in pseudo-code:
```
@let conversionLines = convertText(sample.initCode.argDefaultParams)

@if @conversionLines.empty()
  import six
@

…

def ...
   {@conversionLines}

```

WDYT?

Reply:
Yes this should work. I'll make the change.